p2p/host/hostv2/hostv2.go

```diff
@@ -186,12 +187,11 @@ func (host *HostV2) SendMessage(p p2p.Peer, message []byte) error {
 	if err != nil {
 		logger.Error("NewStream() failed", "peerID", p.PeerID,
 			"protocolID", ProtocolID, "error", err)
-		return fmt.Errorf("NewStream(%v, %v) failed: %v", p.PeerID,
-			ProtocolID, err)
+		return p2p.ErrNewStream
 	}
 	if nw, err := s.Write(message); err != nil {
 		logger.Error("Write() failed", "error", err)
-		return fmt.Errorf("Write() failed: %v", err)
+		return p2p.ErrMsgWrite
 	} else if nw < len(message) {
```

Comment:
It might help to know the peer information where this write message was not successful?

Reply:
sure. added.